models/articles.model.js

```diff
@@ -13,3 +13,17 @@ exports.fetchArticleById = (article_id, next) => {
       next(err);
     });
 };
+
+exports.fetchArticles = () => {
+  return db
+    .query(
+      `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.votes, articles.article_img_url, articles.created_at, COUNT(comments.article_id) AS comment_count 
+      FROM articles
+      LEFT JOIN comments ON articles.article_id = comments.article_id
+      GROUP BY comments.article_id, articles.article_id, articles.*
+      ORDER BY articles.created_at;`
```

Comment:
As we want the articles to be returned in descending order, this SQL query is just missing one word at the end. 🙂